src/test/java/com/wikia/webdriver/elements/mercury/components/discussions/common/BasePostsCreator.java

```diff
@@ -12,6 +13,9 @@
 
   private final CategoryPills categoryPills;
 
+  @Getter
+  private By errorNotification = By.className("error");
```

Comment:
Are you going to use it from outside of this class? If not then `@Getter` might not be necessary.